presto-hive/src/main/java/io/prestosql/plugin/hive/MaterializedViewAlreadyExistsException.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.hive;
+
+import io.prestosql.spi.PrestoException;
+import io.prestosql.spi.connector.SchemaTableName;
+
+import static io.prestosql.spi.StandardErrorCode.ALREADY_EXISTS;
+import static java.lang.String.format;
+
+public class MaterializedViewAlreadyExistsException
```

Comment:
The addition of this class should go in the next commit. Also, should it go in the `presto-iceberg` module (and package) since it's only used by Iceberg?

Reply:
We decided offline to throw a PRESTO_EXCEPTION. I removed this class.